plugins/versionpress/src/Storages/Serialization/SerializedDataToIniConverter.php

```diff
@@ -259,6 +259,12 @@ private static function convertValueToSerializedString($value, $relatedLines = [
     {
         $type = null; // string or number
 
+        //@see IniSerializer::getReplacedEolString
+        $eolReplacement = [
```

Comment:
I think it's not a good idea to have the placeholders duplicated here. What about removing the placeholders before `SerializedDataToIniConverter::fromIniLines`?